lib/membrane_ogg/payloader.ex

```diff
@@ -27,10 +31,18 @@ defmodule Membrane.Ogg.Payloader do
     )
   end
 
+  @spec flush(binary) :: {:ok, state :: binary} | {:error, reason :: atom}
   def flush(native) do
     Native.flush(native)
   end
 
+  @spec make_pages_and_flush(
+          buffer :: binary,
+          native :: binary,
```

Comment:
Most likely it should be
```suggestion
          native :: ref,
```